### DISCUSSION TITLE
ATC updates

### DISCUSSION THREAD

**Discussion by LoreWy** [OHDSI/Vocabulary-v5.0#737] [Q&A] [answered]
Hi all

The Wiki tells me that the ATC vocabulary has a yearly update cycle.
Does someone know about when the update will take place? I am specifically looking forward to finding the changes from these releases in the vocabulary: https://www.whocc.no/atc_ddd_index/updates_included_in_the_atc_ddd_index/

Thanks a lot for all the efforts on keeping vocabularies up to date!

**Comment by mik-ohdsi** [comment, marked as answer]
Hi @LoreWy - apologies for the late reply.
ATC sources are updated yearly and we make an effort to keep up with them. There is an initiative going on for proper and transparent prioritization of vocabulary work, guided by a committee that decides carefully about the sequence of vocabulary refreshes. I understand ATC will be addressed probably within the first half of 2023.
Kind regards - mik
